packages/lab/src/progress/CircularProgress/CircularProgress.css

```diff
@@ -14,41 +16,29 @@
 
 .saltCircularProgress-circle {
   stroke: url("#salt-circular-progress-gradient");
-  stroke-width: 2px;
-}
-
-.saltCircularProgress-small {
-  font-size: 11px;
-}
-
-.saltCircularProgress-medium {
-  font-size: 14px;
-}
-
-.saltCircularProgress-large {
-  font-size: 16px;
 }
 
 .saltCircularProgress-progressValue {
   align-items: center;
   color: var(--salt-text-primary-foreground);
   display: flex;
+  font-size: var(--salt-text-label-fontSize);
+  font-weight: var(--salt-text-label-fontWeight-strong);
   height: 100%;
   justify-content: center;
   left: 0;
   position: absolute;
   width: 100%;
 }
 
-.saltCircularProgress-disabled {
-}
+.saltCircularProgress-disabled {}
```

Comment:
The blue line style looks different in circular and linear,
should you be using `...background-disabled` for the blue line here?